app/build.gradle

```diff
@@ -48,6 +48,7 @@ android {
             dimension "api"
             minSdkVersion 26
             versionNameSuffix "-minApi26"
+            isDefault = true
```

Comment:
what does this do?

Reply:
without it, in Gradle builds, the app is always built and deployed as minApi21 variant (nothing is defined as default). It improves the developer experience by making it easier to debug minApi26 builds. It's annoying to have wrong flavour deployed to emulator, so I added that line.

Reply:
what about Android Studio's Build Variants tab? it works for me lol

Reply:
idk i thought it was for only switching between debug and release mode

Reply:
whatever, with and without this change build for both flavors work